src/debug-server.ts

```diff
@@ -38,6 +77,37 @@ you are, if paused on a breakpoint. Make sure to find and get the contents of an
 Only use continue when ready to move to the next breakpoint. Launch will bring you to the first 
 breakpoint. DO NOT USE CONTINUE TO GET TO THE FIRST BREAKPOINT.`;
 
```

Comment:
We should consider what other value we can bring from https://modelcontextprotocol.io/specification/2025-06-18/server/prompts